src/apps/loyalty/containers/login/index.tsx

```diff
@@ -60,9 +66,69 @@ const StyledOrText = styled.div`
     z-index: -1;
   }
 `
+const ErrorMessage = styled.div`
+  color: ${colors.grayBold};
+  width: 100%;
+  text-align: left;
+  text-transform: capitalize;
+  padding: 10px 0px;
+  border: 1px solid ${colors.redRegular};
+  box-shadow: none;
+  font-size: 15px;
+  text-align: center;
+  background-color: rgba(210, 163, 163, 0.4);
+`
 
 class Login extends React.Component<LoginProps, LoginState> {
+  constructor(props) {
+    super(props)
+    this.state = {
+      email: "",
+      password: "",
+    }
+    this.onSubmit = this.onSubmit.bind(this)
+    this.handleEmailChange = this.handleEmailChange.bind(this)
+    this.handlePasswordChange = this.handlePasswordChange.bind(this)
+  }
+
+  handleEmailChange (e) {
+    this.setState({email: e.target.value})
+  }
+
+  handlePasswordChange (e) {
+    this.setState({password: e.target.value})
+  }
+
+  onSubmit (e) {
+    e.preventDefault()
+    const options = {
+      method: "POST",
+      credentials: "same-origin",
+      headers: {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+      },
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password,
+        _csrf: this.props.form.csrfToken,
+      }),
+    }
+
+    fetch(this.props.form.url, options).then(res => {
+      if (res.status === 200) {
+        // redirect to '/inquiries'
+        window.location.assign(`${sharify.data.BASE_URL}/inquiries`)
+      } else {
+        this.setState({
+          error: decodeURI(res.url.split("error=")[1]),
+        })
+      }
+    })
```

Comment:
this should probably `catch` also, maybe just log the error (or even maybe put it in the error bar too?)